examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java

```diff
@@ -179,18 +183,35 @@ private static void initTracing() {
   public static void main(String[] args) throws Exception {
     initTracing();
     initStatsExporter();
-    ServletContextHandler context = new ServletContextHandler(ServletContextHandler.SESSIONS);
-    context.setContextPath("/");
 
     Server server = new Server(8080);
-    ServletHandler handler = new ServletHandler();
-    server.setHandler(handler);
-
-    handler.addFilterWithMapping(
-        OcHttpServletFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
-    handler.addServletWithMapping(HelloServlet.class, "/*");
-
-    server.start();
-    server.join();
+    ServletContextHandler contextHandler =
+        new ServletContextHandler(ServletContextHandler.SESSIONS);
+    contextHandler.setContextPath("/helloworld");
+    ServletHolder sh = new ServletHolder(new HelloServlet());
+    contextHandler.addServlet(sh, "/request/*");
+
+    // Enable tracing by adding OcHttpServleFilter for all path
+    contextHandler.addFilter(OcHttpServletFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
+
+    // Uncomment following line to use B3Format for trace context propagation.
+    // contextHandler.setAttribute(
+    //    OC_TRACE_PROPAGATOR, Tracing.getPropagationComponent().getB3Format());
+
+    // By default publicEndpoint parameter is set to false and incoming trace context is added as
```

Comment:
I may be misreading this but it sounds like the incoming trace context is added as a parent link in both the true and false cases.

Reply:
you are right. the first one should be parent.